### DISCUSSION TITLE
CharmLlama: a cli application for Ollama

### DISCUSSION THREAD

**Discussion by marpit19** [charmbracelet/bubbletea#1089] [Show and tell]
CharmLlama is a delightful CLI-based chat application that allows you to interact with Ollama's open-source language models locally. Built with Go and Charm libraries, it offers a rich terminal experience for AI-powered conversations.

Here is the link: https://github.com/marpit19/charmlama

![output](https://github.com/user-attachments/assets/1b13a6aa-a55c-4cd4-8e6e-4163658e3d27)

**Comment by Delta456** [comment]
Looks nice! Feel free to share at [Charm Community](https://github.com/orgs/charm-community/discussions/new/choose) too

**Reply by marpit19** [reply]
Thanks for the feedback, sure will do it!
